src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,29 @@
+$cell-size: 100px;
+$font-size: 30px;
+$cell-distance: 3px;
+$padding-calendar: 10px;
+$amount-day: 7;
+$calendar-size: $amount-day * $cell-size + $cell-distance * ($amount-day - 1);
+$distance-cell-left: $cell-size + $cell-distance;
+
+$days:
+  "mon" 0,
+  "tue" $distance-cell-left,
+  "wed" (2 * $distance-cell-left),
+  "thu" (3 * $distance-cell-left),
+  "fri" (4 * $distance-cell-left),
+  "sat" (5 * $distance-cell-left),
+  "sun" (6 * $distance-cell-left);
```

Comment:
it's better to calculate values in loop ($index * $distance-cell-left)

Reply:
thanks. done